packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -15,10 +15,14 @@
 
 Name: katello-host-tools
 Version: 3.5.7
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: A set of commands and yum plugins that support a Katello host
 Group:   Development/Languages
+%if 0%{?suse_version}
+License: LGPL-2.0
```

Comment:
why is suse using a different notation than everyone else?

Reply:
"I can not answer this question"
It did not build on Open Suse Build Service because of the license. Then I remembered:
https://github.com/candlepin/subscription-manager/blob/0557699a9b65a77812604e7038fc487f100675b0/subscription-manager.spec#L117

Reply:
:vomiting_face: